caom2/src/test/java/ca/nrc/cadc/caom2/types/PolygonTest.java

```diff
@@ -177,4 +177,64 @@ public void testInvalidPolygon()
         }
     }
     
+    @Test
+    public void testValidateSegments()
+    {
+        try
+        {
+            MultiPolygon validMP = new MultiPolygon();
```

Comment:
Why did you go backwards here instead of adding Points to poly as in the next test case?

Reply:
The MultiPolygon is immutable in this test and I am just validating the initial poly (line 199) before I make the outer polygon invalid in various ways. Polygon.validate() calls samples.validate() and I want to be sure the MP is valid.